viadot/sources/outlook.py

```diff
@@ -4,9 +4,9 @@
 import pandas as pd
 from O365 import Account
 
-from ..config import local_config
-from ..exceptions import CredentialError
-from .base import Source
+from viadot.config import local_config
+from viadot.exceptions import CredentialError
+from viadot.sources.base import Source
```

Comment:
Were these changes necessary?

Reply:
they are not, but I added them here to standardize the import way. Better absolute import instead of relative one. In any case, this doesn't affect to the code.